rwengine/src/objects/CharacterObject.cpp

```diff
@@ -573,6 +573,8 @@ void CharacterObject::removeFromInventory(int slot) {
 }
 
 void CharacterObject::cycleInventory(bool up) {
+    if (currentVehicle != nullptr) return;
```

Comment:
Maybe this should happen when calling this function instead of in this function?
I'd assume character functions to work at all times, however, input in cars should be treated differently and that processing is probably done where this is called from.